### PR TITLE
Fix #1068: brake influences NWS and the nose wheel steering is centered if the strut is uncompressed

### LINKED ISSUE #1068: Nose gear should be disconnected to rudder pedal when airborne
I read a book that shows

> During flight the nose wheel extends fully, bringing a lock mechanism into place which hold the nose wheel center and free from rudder pedal action.

So I convince that the nose gear should not controllable when airborne.

### PR DESCRIPTION
**Pull request by legoboyvdlp**
Fixes #1068 

Ready for merging... I hope?

### CODE REVIEW

**Comment by wlbragg** [comment]
@legoboyvdlp Other than @dany93 comments I really have nothing. it works great. I was a bit curious what happens when you turn left and brake right. Seems turn right takes precedence which I guess it should.
My only other question was concerning this block...
~~~
<switch name="fcs/brake-influence-enabled"> <!-- brake should not influence nws if the aircraft is not moving -->
                <default value="0"/>
                <test logic="OR" value="1">
                    /gear/gear[0]/rollspeed-ms gt 0.5
                </test>
            </switch>
~~~

I didn't totally understand the mechanics here, how exactly the mechanism allows this functionality.

1) We know this to be true that the wheel has to be moving for the break to allow this extra 20deg movement?
2) How did you come up with .5 meters per second? Why not .01 or even less?

**Comment by legoboyvdlp** [comment]
Well, if there is no forward motion, I think there would be no side force at all - but maybe 0.1 m/s is better?

**Comment by gilbertohasnofb** [comment]
@legoboyvdlp Really good work, lego, this is working really well!

**Comment by gilbertohasnofb** [comment]
Is this ready to be merged?

**Comment by dany93** [comment]
In my opinion, yes for the code and functioning, but what about my remarks and questions about the form?

**Comment by gilbertohasnofb** [comment]
@dany93 sorry, I did not realize those hadn't been addressed. @legoboyvdlp could you take a look at those when you have a chance?

File: c172p.xml [hunk 1]
```diff
@@ -841,6 +841,97 @@
...
+                </function>
+            </fcs_function>
+
+            <summer name="fcs/brake-sum">
+                <input>fcs/nose-gear-right-brake-steer-factor</input>
+                <input>fcs/nose-gear-left-brake-steer-factor</input>
+                <clipto>
+                    <min>-1</min>
+                    <max>1</max>
+                </clipto>
```
[the remark this answers is not in the document]
Comment (legoboyvdlp) [reply c172p.xml]: Yes.

File: c172p.xml [hunk 2]
```diff
@@ -841,6 +841,97 @@
...
+                <function>
+                    <product>
+                        <property>fcs/brake-influence-enabled</property>
+                        <property>fcs/brake-sum</property>
+                    </product>
+                </function>
+                <clipto>
+                    <min>-1</min>
+                    <max>1</max>
+                </clipto>
```
[the remark this answers is not in the document]
Comment (legoboyvdlp) [reply c172p.xml]: no, removed

File: c172p.xml [hunk 3]
```diff
@@ -841,6 +841,97 @@
...
+                    <table>
+                        <independentVar lookup="row">fcs/brake-sum-adjusted</independentVar>
+                        <independentVar lookup="column">/controls/flight/rudder</independentVar>
+                        <tableData>
+                          -1        0   1
+                          -1    -30 0   10 
+                           0    -10 0   10
+                           1    -10 0   30  
+                        </tableData>
+                    </table>
```
[the remark this answers is not in the document]
Comment (legoboyvdlp) [reply c172p.xml]: done

File: c172p.xml [hunk 4]
```diff
@@ -841,6 +841,97 @@
...
+            </fcs_function>
+            
+            <actuator name="fcs/steer-deg">
+                <input>fcs/nws-cmd-deg-adjusted</input>
+                <rate_limit>80</rate_limit> <!-- 80 deg / s , or 1/8 second to full deflection with NWS -->
+                <output>fcs/steer-pos-deg</output>
+                <clipto>
+                    <min>-30</min>
+                    <max>30</max>
+                </clipto>
```
[the remark this answers is not in the document]
Comment (legoboyvdlp) [reply c172p.xml]: no, removed

**Comment by legoboyvdlp** [comment]
I reduced the minimum speed to 0.25 m/s - if anyone wants that can be reduced further, if not this can definitely be merged.

File: c172p.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dany93) [reply c172p.xml]: Sorry if that's me, but your response is too short.
`nose-gear-right-brake-steer-factor` goes from 0 to 1,
`nose-gear-left-brake-steer-factor` goes from 0 to -1.
How can their sum go beyond the -1 to +1 range?

**Comment by dany93** [comment]
Still a question about `<clipto>`.

File: c172p.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (legoboyvdlp) [reply c172p.xml]: If fcs/X-brake-cmd-norm is ever greater than 1, then the sum will go beyond that range, but I'll remove that anyway, assuming that will never go beyond limits.

**Comment by wlbragg** [comment]
@dany93 if this answers all your comments let me know and I will merge or you can.

**Comment by dany93** [comment]
Yes, IMO that's ready to be merged.

File: c172p.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dany93) [reply c172p.xml]: `fcs/X-brake-cmd-norm` should never be greater than 1 (from the definition of 'normalized').

**Comment by wlbragg** [comment]
Thank you @legoboyvdlp

**Comment by dany93** [comment]
Thank you too. Nice work.
And I hope not having been unpleasant.

**Pull request merged**